fastparquet/test/test_api.py

```diff
@@ -430,4 +446,5 @@ def test_compression(tempdir):
     assert (False in set(df2["x"] == df["x"])) is False
     assert (False in set(df2["y"] == df["y"])) is False
     assert (False in set(df2["z"] == df["z"])) is False
+    assert (False in set(df2["w"] == df["w"])) is False
```

Comment:
Does `pandas.testing.assert_frame_equal` work instead of the four lines here? Alternatively, could do `for col in 'xyzw':`